### ISSUE TITLE
Move more tests here

### ISSUE THREAD

**Issue by wagenet** [graphiti-api/graphiti-rails#75]
Right now, most tests live in the main graphiti repo. We should move the graphiti-rails ones here.

**Comment by richmolj** [comment]
I agree conceptually, but particularly for integration specs I'm not sure. Seems like we could get into a situation where an update to graphiti codebase breaks the overall behavior users expect, but we don't know about it because the PR ran specs for graphiti, not graphiti+graphiti-rails

**Comment by wagenet** [comment]
@richmolj we’d probably want some on both sides ideally.

**Comment by jkeen** [comment]
These satellite gems are getting absorbed into the graphiti gem for 2.0 release to make things easier. ~85% of graphiti users are on rails, and it won't break people who aren't.
